facebook-commerce.php

```diff
@@ -3753,129 +3546,6 @@ function ajax_display_test_result() {
 		wp_die();
 	}
 
-
-	/**
-	 * Schedules a recurring event to sync products.
-	 *
-	 * @deprecated 1.10.0
-	 */
-	function ajax_schedule_force_resync() {
-
-		wc_deprecated_function( __METHOD__, '1.10.0' );
-		die;
-	}
-
-
-	/**
-	 * Adds an recurring action to sync products.
-	 *
-	 * The action is scheduled using a cron schedule instead of a recurring interval (see https://en.wikipedia.org/wiki/Cron#Overview).
-	 * A cron schedule should allow for the action to run roughly at the same time every day regardless of the duration of the task.
-	 *
-	 * @since 1.10.0
-	 *
-	 * @param int $offset number of seconds since the beginning of the daay
-	 */
-	public function schedule_resync( $offset ) {
```

Comment:
The method is being used in

https://github.com/woocommerce/facebook-for-woocommerce/blob/bf97a40040324359069f4d0190d932e495987923/includes/Lifecycle.php#L191-L193